docker/rust.docker

```diff
@@ -1,28 +1,38 @@
 FROM codewars/base-runner
 
-# Install Rust Compiler
-RUN curl -sSf https://static.rust-lang.org/rustup.sh | sh
+# Install Rust v1.15.1 to /usr/local
+RUN set -ex \
+ && cd /tmp \
+ && curl -fSsL https://static.rust-lang.org/dist/2017-02-08/rust-1.15.1-x86_64-unknown-linux-gnu.tar.gz -o rust.tar.gz \
```

Comment:
Thanks! This is definitely better. I didn't read the `rustup` documentation enough. I'll take a look at your PR.